src/App.js

```diff
@@ -114,18 +114,8 @@ class App extends Component {
   render() {
     
     //const headerBackground = require("./images/headerbackground.png")
-    const backGroundUrl = require("./images/fancy-pants.jpg")
+    
     const flixieTitle = require('./images/fliXie_dark.png')
```

Comment:
I was wondering actually if this is best practice for importing images. It doesn't seem to be very straightforward, but you didn't change it or comment it. Can I assume it's normal?